extension/src/cli/dvc/runner.ts

```diff
@@ -19,8 +19,7 @@ import { Disposable } from '../../class/dispose'
 
 export const autoRegisteredCommands = {
   EXPERIMENT_RESET_AND_RUN: 'runExperimentReset',
-  EXPERIMENT_RUN: 'runExperiment',
-  EXPERIMENT_RUN_QUEUED: 'runExperimentQueue'
```

Comment:
[F] We've move this from here into the executor (no output is returned to the runner's terminal anymore anyway).

Reply:
The legacy behaviour was for all of the logs to be fed into the terminal's output.